packages/protocol/RecordedEventsCache.ts

```diff
@@ -1,8 +1,59 @@
-import { KeyboardEvent, MouseEvent, NavigationEvent } from "@replayio/protocol";
+import {
+  KeyboardEvent,
+  MouseEvent,
+  NavigationEvent as ProtocolNavigationEvent,
+} from "@replayio/protocol";
 import { createSingleEntryCache } from "suspense";
 
+import { compareNumericStrings } from "protocol/utils";
 import { replayClient } from "shared/client/ReplayClientContext";
 
+// TODO [BAC-4618] Update the type definition in the protocol package
+// This hack dates back to 2021 (github.com/replayio/devtools/pull/4713)
+export type NavigationEvent = Omit<ProtocolNavigationEvent, "kind"> & {
+  kind: "navigation";
+};
```

Comment:
This is a long-standing issue but I filed BAC-4618 to hopefully address it better than this